README.md

```diff
@@ -90,15 +90,15 @@ resource "aws_s3_bucket" "lambda_deploy" {
 }
 
 module "my_lambda" {
-  source         = "github.com/instructure/tf_versioned_lambda//modules/node"
+  source         = "github.com/instructure/tf_versioned_lambda/modules/node"
```

Comment:
This actually isn't correct, look at the github section here: https://www.terraform.io/docs/modules/sources.html, the double slash does magic